src/ui/Views/Shared/_Layout.cshtml

```diff
@@ -39,15 +41,15 @@
                 <ul id="proposition-links">
                     @if (User.Identity.IsAuthenticated)
                     {
-                        <li><a asp-controller="Auth" asp-action="Logout">Sign out</a></li>
+                    <li><a asp-controller="Auth" asp-action="Logout">Sign out</a> ( <a href="@profileUrl"> @name </a> ) </li>
```

Comment:
Can you drop the spaces around the link?

eg `(name)` not `( name )`

Reply:
Like this

![image](https://user-images.githubusercontent.com/470137/42389141-ac292708-813f-11e8-908f-3787ad44e894.png)